### PR TITLE
i18n(es): Updated `prefetch.mdx` & `configuration-reference.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `prefetch.mdx`

`prefetch.mdx`
* [`f6563bb`](https://github.com/withastro/docs/commit/f6563bbd815168345e402fd56204013797c262e8)

`configuration-reference.mdx`
* [`e724d5e`](https://github.com/withastro/docs/commit/e724d5ed5821e888f56d64113bebe330dd6c28a8)

### CODE REVIEW

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 1]
```diff
@@ -510,6 +510,26 @@ del directorio `src/pages`.
...
+<p>
+
+**Tipo:** `boolean`<br />
+**Por defecto:** {false}<br />
+<Since v="2.8.0" />
+</p>
+
+Define si el código del middleware SSR se incluirá o no al compilar.
+
+Cuando está habilitado, el código del middleware no se incluye y se importa en todas las páginas durante la compilación. Para ejecutar e importar manualmente el código del middleware, establece `build.excludeMiddleware: true`:
```
Comment (dreyfus92) [inline src/content/docs/es/reference/configuration-reference.mdx:524]:
```suggestion
Cuando está habilitado, el código del middleware no es empaquetado ni se importa en todas las páginas durante la compilación. Para ejecutar e importar manualmente el código del middleware, establece `build.excludeMiddleware: true`:
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59, te dejo una sugerencia 😄

**Comment by Waxer59** [comment]
Gracias por la revisión 🤠

**Review by Yan-Thomas — APPROVED** [review]
Thanks peeps, I noticed a small issue in the Configuration Reference I'll fix upstream and let you know to update later!

**Pull request merged**